edgeboxctl.service

```diff
@@ -0,0 +1,29 @@
+[Unit]
+Description=Edgebox Control Module Service
+ConditionPathExists=/home/system/

```

Comment:
Maybe we can remove this line. Is a good check for the raspberry pi and DO images, but it will make it not work with any system that does not have this `system` user created :/